color-eyre/tests/data/theme_error_control_minimal.txt

```diff
@@ -7,17 +7,17 @@ Error:
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
   [96m                              ⋮ 5 frames hidden ⋮                               [0m
    6: [91mtheme::get_error::create_report[0m[90m::h43540daddae98383[0m
-      at [35m/home/freja/dev/rust/eyre/color-eyre/tests/theme.rs[0m:[35m17[0m
+      at [35m/home/username/dev/rust/eyre/color-eyre/tests/theme.rs[0m:[35m17[0m
```

Comment:
How does this work? I see you replaced your handle with the string 'username' -- is that the configuration that we need to maintain now for our tester VMs?

Reply:
spoke in a call -- this is placeholder text that is line-sensitive, the colors are what matters.